deps/wazuh_testing/wazuh_testing/tools/file.py

```diff
@@ -281,6 +281,30 @@ def remove_file(file_path):
             delete_path_recursively(file_path)
 
 
+def copy_files_in_folder(folder, target_folder='/tmp', files_to_move=None):
```

Comment:
```suggestion
def copy_files_in_folder(src_folder, dst_folder='/tmp', all_files=True):
```

Reply:
Applied change in https://github.com/wazuh/wazuh-qa/pull/3566/commits/ec2169488b0e11665c776cab45c3a631106364f7.

Only removed the `target_folder` folder. The `files_to_move` variable allows to pass a list of files that need to be moved in case the folder has some files that need to be moved and some to be left in the folder. This way, a value of `None` moves all files in the folder, and if a list is passed, only files in the list are moved.